core/src/layers/prometheus_client.rs

```diff
@@ -107,6 +107,7 @@ pub struct PrometheusClientLayerBuilder {
     entries_rate_buckets: Vec<f64>,
     duration_seconds_buckets: Vec<f64>,
     ttfb_buckets: Vec<f64>,
+    enable_high_cardinality_labels: bool,
```

Comment:
Providing an `enable_high_cardinality_labels` option doesn't make sense to me. I think we should introduce a new configuration called `disable_label_root`.

Users who manage multiple operators, each with a different root, can disable the `root` label if they don't care about the metrics over vary roots.

Reply:
how about calling it `enable_label_root`? i forgot which material i had once read that it's better to use 'positive' phrasing in options 🤔

Reply:
My concern is that adding `disable_label_root` is additive and can be considered a compatible change, whereas `enable_label_root` would break all existing users.

Reply:
we can default it as `true` to avoid introduce any breaking change, wdyt?

Reply:
> we can default it as `true` to avoid introduce any breaking change, wdyt?

I feel offering a `disable_label_root` option, rather than an `enable_label_root` defaulting to `true`, would be easier to understand.

Reply:
ok, let me follow this path

Reply:
fixed in a401eb1b63, PTAL

Reply:
> ok, let me follow this path

Thank you for this.